android/res/layout/menu_buttons_line.xml

```diff
@@ -14,46 +16,50 @@
       android:id="@+id/help"
       style="@style/MwmTheme.Menu.Line.Button"
       android:src="@drawable/ic_question_mark"
-      android:tag="@string/tag_menu_collapse"/>
+      android:tag="@string/tag_menu_collapse" />
 
     <Space
       android:layout_width="0dp"
       android:layout_height="match_parent"
-      android:layout_weight="1"/>
+      android:layout_weight="1" />
 
     <ImageView
       android:id="@+id/search"
       style="@style/MwmTheme.Menu.Line.Button"
       android:src="@drawable/ic_menu_search"
-      android:tag="@string/tag_menu_collapse"/>
+      android:tag="@string/tag_menu_collapse" />
 
     <Space
       android:layout_width="0dp"
       android:layout_height="match_parent"
-      android:layout_weight="1"/>
+      android:layout_weight="1" />
```

Comment:
It doesn't make sense to insert spaces here. Let's leave the old formatting for a cleaner history.